ui/components/UserPreferences.js

```diff
@@ -300,7 +324,38 @@ class UserPreference extends React.Component {
             <MesherySettingsPerformanceComponent />
           }
           {tabVal == 2 && userPrefs && providerType != 'local' &&
-            <ExtensionSandbox type="user_prefs" Extension={(url) => RemoteUserPref({ startOnZoom, handleToggle, url })} />
+            <div>
```

Comment:
@utkarsh-pro this should be an empty `div`, yes? The contents of this extension point should come from the connected remote provider, if that remote provider offers this capability.

Reply:
@leecalcote, yes. @NikhilSharma03 I think the changes that are done here should rather be done on the provider side.

Reply:
I see that a table container is added, is there some issue that you faced while adding this to the extension @NikhilSharma03?

Reply:
> @leecalcote, yes. @NikhilSharma03 I think the changes that are done here should rather be done on the provider side.

Does provider side mean here :sweat_smile:  https://github.com/meshery/meshery/blob/a541cda0a171c470a75f78ef736f7427f5c6c8f8/ui/components/NavigatorExtension.js